.circleci/config.yml

```
@@ -48,6 +48,14 @@ jobs:
       - install-python
       - run: bazel test //tests/end_to_end/... --test_output=streamed --spawn_strategy=standalone --python_version PY3 --python_path $(which python)
 
+  test-markdown-link-health:
+    machine: true
+    working_directory: ~/kglib
+    steps:
+      - checkout
+      - run: npm install -g markdown-link-check
+      - run: find . -name \*.md | xargs -L1 markdown-link-check -v
```

@jmsfltchr 

```suggestion
      - run: find . -name \*.md | xargs -L1 npx markdown-link-check -v
```

and then you don't need to `npm install` it